baseapp/abci.go

```diff
@@ -571,6 +573,7 @@ func (app *BaseApp) ExtendVote(_ context.Context, req *abci.RequestExtendVote) (
 		WithHeaderInfo(coreheader.Info{
 			ChainID: app.chainID,
 			Height:  req.Height,
+			Hash:    req.Hash,
```

Comment:
Added this given that it might be needed

`The header hash of the proposed block that the vote extension is to refer to.`